crates/ruff_python_parser/src/lexer.rs

```diff
@@ -429,6 +429,31 @@ impl<'source> Lexer<'source> {
                     value.push('\\');
                 }
                 '\n' | '\r' | EOF_CHAR => {
+                    let mut chars = value.chars().rev();
+                    // Check if it's actually a help magic command. The help magic
+                    // command takes priority over any other magic command.
```

Comment:
```suggestion
                    // Check if it's a help magic command. The help magic
                    // command takes priority over any other magic command.
```

What I understand here is that it could now be either, right?

Reply:
I think this comment is a bit unclear. The official terminology for the help command is "Dynamic object introspection". I'm trying to say if `%foo?` is actually a help command (`?`) (which it is). `?` takes priority over any other magic tokens. So, if we're lexing a magic command (as it started with a magic token) but it ended with either `?` or `??`, then the latter takes precedence.